fire/cli/ts/__init__.py

```diff
@@ -9,4 +26,130 @@ def ts():
     pass
 
 
+def _print_tidsserieoversigt(tidsserieklasse: type, punkt: Punkt = None):
```

Comment:
Du kan godt være mere explicit med typen her:

```
def _print_tidsserieoversigt(tidsserieklasse: type[Tidsserie], punkt: Punkt = None):
```

eller måske skal det være `Type[Tidsserie]`, det kan jeg ikke lige huske i hovedet. `Type` kan importeres fra `typing`

Reply:
Gør sig også gældende for de to funktioner herunder: `_find_tisserie()` og `_udtræk_tidsserie()`.

Reply:
Ok, var ikke klar over man kunne lave det sådan! Det virker med type